turnkey-version.py

```diff
@@ -1,52 +1,54 @@
-#!/usr/bin/python
+#! /usr/bin/python
 # Ad-hoc version detection for a system installed from a TurnKey
 # appliance. This is a transitory package so we can identify the versions
 # in current appliance which were designed before we needed this for
 # tklbam. In the future it shouldn't be needed as all appliances will be
 # marked with their versions.
 
+from __future__ import print_function
+from builtins import str
```

Comment:
You should be able to rewrite each usage without this and it will be clearer anyway.. Also, `str` will do different things under Python 2 and and 3 I fear... ;)

Reply:
These lines were added by `futurize`, not by me.  If you remove them, I believe you lose compatibility with Python2.

Reply:
Sure, I was really suggesting we remove/fix the stuff (see below) that requires it. :)